src/www/ui/page/AdminLicenseCandidate.php

```diff
@@ -262,6 +279,32 @@ private function mergeCandidate($candidate, $suggest, $vars)
     return true;
   }
 
+  protected function doDeleteCandidate($rfPk)
+  {
+    $dbManager = $this->getObject('db.manager');
+    $stmt = __METHOD__.".$rfPk";
+    $dbManager->prepare($stmt, "SELECT DISTINCT(uploadtree_fk) FROM clearing_event "
+                               ."WHERE rf_fk=$1 AND removed=false AND uploadtree_fk NOT IN "
+                               ."(SELECT DISTINCT(uploadtree_fk) FROM clearing_event WHERE "
+                               ."rf_fk=$1 AND removed=TRUE);");
+    $result = $dbManager->execute($stmt, array($rfPk));
+    $dataFetch = $dbManager->fetchAll($result);
+    $dbManager->freeResult($result);
+    if(empty($dataFetch)){
+      $dbManager->getSingleRow('DELETE FROM license_candidate WHERE rf_pk=$1', array($rfPk), __METHOD__.".delete");
+      return new Response('true', Response::HTTP_OK, array('Content-type'=>'text/plain'));
+    }else{
+      $treeDao = $this->getObject('dao.tree');
+      $Path  = "Cannot remove license, because it is in use at \n\n";
```

Comment:
The variable name `$Path` is confusing here, and please start variables with lowercase.